pwa/src/components/componentCard/ComponentCard.module.css

```diff
@@ -88,7 +94,9 @@
 
 .servicesCategory,
 .serviceCategory {
-  --conduction-tag-background-color: var(--web-app-layer-category-color-services);
+  --conduction-tag-background-color: var(
+    --web-app-layer-category-color-services
+  );
```

Comment:
All these changes are prettier changes, which shouldn't be happening. Can you check that you're using the `.prettierrc` file from within this repository and then re-save this file?